src/Service/SudoModeService.php

```diff
@@ -0,0 +1,51 @@
+<?php declare(strict_types=1);
+
+namespace SilverStripe\SecurityExtensions\Service;
+
+use SilverStripe\Control\Session;
+use SilverStripe\Core\Config\Configurable;
+use SilverStripe\ORM\FieldType\DBDatetime;
+
+class SudoModeService implements SudoModeServiceInterface
+{
+    use Configurable;
+
+    /**
+     * The lifetime that sudo mode authorization lasts for, in minutes
```

Comment:
I think we should document that this isn't necessarily going to actually last longer than a session - We could also link to the configuration for that `SilverStripe\Control\Session::$timeout`.